scheduler/src/cook/components.clj

```diff
@@ -261,11 +306,22 @@
    :exception-handler (fnk [] ((lazy-load-var 'cook.util/install-email-on-exception-handler))) ;;TODO parameterize
    })
 
-(def config-settings
-  "Parses the settings out of a config file"
+(def parse-settings
+  "Parses the raw settings out of a config file. Returns a
+  map (implemented as a Plumatic Plumbing graph) of various cooked
+  settings derived from the raw config file EDN."
   (graph/eager-compile
     {:server-port (fnk [[:config port]]
                        port)
+     :user-privileges (fnk [[:config {user-privileges {}}]]
+                           user-privileges)
+
+     :authorization-config (fnk [[:config authorization-config]]
+                                authorization-config)
+
+     :admins (fnk [[:config [:authorization-config admins]]]
```

Comment:
That got removed on the shares-rest-endpoints branch in a different commit than the one I merged back (in 82458ac780707bcd000a342383b5679a65a486c7).
I've just cherry-picked it back into this branch. Sorry about the confusion.